hw/ip/gpio/data/gpio.prj.hjson

```diff
@@ -18,7 +21,7 @@
         life_stage:         "L1",
         design_stage:       "D2",
         verification_stage: "V2",
-        commit_id:          "f3039d7006ca8ebd45ae0b52b22864983876175d",
+        // commit_id:          "f3039d7006ca8ebd45ae0b52b22864983876175d",
```

Comment:
@eunchan @sjgitty 
Should the "WIP" revision indicate the commit id?
If the commit ID is present, the D# and V# get linked to the github committed version (that's how I have updated the tooling), which means they are no longer current. My feeling is we only set the commit id after we have signed off.

Reply:
I think we should only include the commit_id for signed off designs

Reply:
I also updated the tool such that it ignores the commit id for the latest revision and shows the html version of the checklist. For all previous revisions, it will point to the committed version in GitHub.

Reply:
Can you check those signed-off links? I'm finding that some of them 
(maybe all of them?) don't work, what they're pointing to is content that
still says "In Progress". Not sure if that is the fault of the commit ID or of
the .py script

Reply:
This should be fixed now.